### PR TITLE
[AIC-py][editor] bundle easy changes

### PR DESCRIPTION
**Pull request by jonathanlastmileai**
[AIC-py][editor] bundle easy changes

- make aiconfig runtime
- log to file
- make path relative to aiconfig root

---
Stack created with [Sapling](https://sapling-scm.com). Best reviewed with [ReviewStack](https://reviewstack.dev/lastmile-ai/aiconfig/pull/556).
* #567
* #565
* #563
* #560
* __->__ #556
* #555

### CODE REVIEW

**Review by rholinshead — APPROVED** [review]

File: python/src/aiconfig/editor/server/server.py [hunk 1]
```diff
@@ -26,6 +34,7 @@ class EditConfig(core_utils.Record):
 @dataclass
 class ServerState:
     count = 0
+    aiconfig_runtime: AIConfigRuntime | None = None
```
Comment (rossdanlm) [inline python/src/aiconfig/editor/server/server.py:37]: offline discussion note: need to lock

File: python/src/aiconfig/scripts/aiconfig_cli.py [hunk 2]
```diff
@@ -67,8 +67,9 @@ def _process_cli_config(cli_config: AIConfigCLIConfig) -> Result[bool, str]:
 def _run_frontend_server_background() -> Result[list[subprocess.Popen[bytes]], str]:
     LOGGER.info("Running frontend server in background")
     try:
-        p1 = subprocess.Popen(["yarn"], cwd="src/aiconfig/editor/client")
-        p2 = subprocess.Popen(["yarn", "start"], cwd="src/aiconfig/editor/client")
+        p1 = subprocess.Popen(["yarn"], cwd="python/src/aiconfig/editor/client")
+        p2 = subprocess.Popen(["yarn", "start"], cwd="python/src/aiconfig/editor/client", stdin=subprocess.PIPE)
```
Comment (rossdanlm) [inline python/src/aiconfig/scripts/aiconfig_cli.py:71]: Add comment TODO to run from `aiconfig` root dir

**Review by rossdanlm — APPROVED** [review]
thanks, looks great, maybe add test plan of video in comments!

File: python/src/aiconfig/scripts/aiconfig_cli.py (hunk 2 above)
Comment (jonathanlastmileai) [reply python/src/aiconfig/scripts/aiconfig_cli.py:71]: ?

**Pull request merged**